themes/src/main/resources/theme/keycloak.v2/account/src/app/content/ContentPage.tsx

```diff
@@ -40,27 +40,46 @@ export class ContentPage extends React.Component<ContentPageProps> {
     public render(): React.ReactNode {
         return (
             <React.Fragment>
-                <ContentAlert/>
-                <section id="page-heading" className="pf-c-page__main-section pf-m-light">
-                    <Grid>
-                        <GridItem span={11}><Title headingLevel='h1' size='3xl'><strong><Msg msgKey={this.props.title}/></strong></Title></GridItem>
-                        {this.props.onRefresh &&
-                            <GridItem span={1}>
-                                <Tooltip content={<Msg msgKey='refreshPage'/>}>
-                                    <Button aria-describedby="refresh page" id='refresh-page' variant='plain' onClick={this.props.onRefresh}>
-                                        <RedoIcon size='sm'/>
-                                    </Button>
-                                </Tooltip>
-                            </GridItem>
-                        }
-                        {this.props.introMessage && <GridItem span={12}> <Msg msgKey={this.props.introMessage}/></GridItem>}
-                    </Grid>
-                </section>
+            <ContentAlert />
 
-                <section className="pf-c-page__main-section pf-m-no-padding-mobile">
-                    {this.props.children}
-                </section>
-            </React.Fragment>
+            <PageSection variant={PageSectionVariants.light} className="pf-u-pb-xs">
+              <Split>
+                <SplitItem isFilled>
+                  <TextContent>
+                    <Title headingLevel="h1" size="2xl" className="pf-u-mb-xl">
+                      <Msg msgKey={this.props.title} />
+                    </Title>
+                    {this.props.introMessage && (
+                      <Text component="p">
+                        <Msg msgKey={this.props.introMessage} />
+                      </Text>
+                    )}
+                  </TextContent>
+                </SplitItem>
+                {this.props.onRefresh && (
+                  <SplitItem>
+                    <Tooltip content={<Msg msgKey="refreshPage" />}>
+                      <Button
+                        aria-describedby="refresh page"
+                        id="refresh-page"
+                        variant="link"
+                        onClick={this.props.onRefresh}
+                        icon={<SyncAltIcon />}
+                      >
+                        Refresh
+                      </Button>
+                    </Tooltip>
+                  </SplitItem>
+                )}
+              </Split>
+            </PageSection>
+
+            {/* TODO: this might be better left here, but down the road you'd probably want a parameter for variant and whether there's padding or not */}
+            {/* If so, the page sections in the children should be removed */}
+            {/* <PageSection isFilled variant={PageSectionVariants.light}> */}
```

Comment:
Not a fan of leaving code commented out like this, can we remove it instead?

Reply:
Yes! Cleaning it up